### ISSUE TITLE
about godot 3d->2d projection

### ISSUE THREAD

**Issue by gu-fan** [Zireael07/Space-Frontier#5]
in this issue https://github.com/godotengine/godot-proposals/issues/4311
your post code there.
I didn't understand what's the `g_transform`, is it the global transform?

**Comment by Zireael07** [comment]
Yes, but that question would be better asked on the proposal itself (this repo has nothing to do with that proposal)

**Comment by gu-fan** [comment]
haha, I thought I should not bothering in that thread but seems it auto linked by github
